packages/atomic/src/components/search/atomic-refine-modal/atomic-refine-modal.tsx

```diff
@@ -187,12 +187,6 @@ export class AtomicRefineModal implements InitializableComponent {
   }
 
   private renderFilters() {
-    const isFacetElements = this.bindings.store.getFacetElements().length > 0;
-
-    if (!isFacetElements) {
-      return;
-    }
-
```

Comment:
I think this was added just to improve performance. Everything works fine without it.

Reply:
ah yes, you are right ! I completely missed that 😅